kaggle_environments/envs/halite/halite.js

```diff
@@ -358,56 +413,42 @@ async function renderer({
     bgCtx.fillStyle = bgStyle;
     bgCtx.fillRect(0, 0, bgCanvas.width, bgCanvas.height);
 
-    // Render the halite.
-    halite.forEach((cellHalite, pos) => {
-      if (!cellHalite) return;
-
-      let { dx, dy, ds, ss } = getCoords(pos);
-      let sx = 0;
-      let sy = 0;
-
-      const pct = Math.min(cellHalite, maxCellHalite) / maxCellHalite;
-      let scale = 1;
-
-      // Scale by the halite size.
-      if (pct > 0.7) {
-        scale = pct;
-      } else if (pct < 0.3) {
-        scale = pct / 0.3;
-        sy = 200;
-      } else {
-        sy = 100;
-        scale = pct + 0.3;
-      }
+    // Render bounding box.
+    bgCtx.strokeStyle = "white";
+    bgCtx.lineWidth = 0.5;
+    bgCtx.strokeRect(
+      topLeftCell.dx - boxPadding,
+      topLeftCell.dy - boxPadding,
+      botRightCell.dx + botRightCell.ds - topLeftCell.dx + 2 * boxPadding,
+      botRightCell.dy + botRightCell.ds - topLeftCell.dy + 2 * boxPadding);
 
-      // Apply the scale.
-      scale = Math.max(0.3, scale);
-      dx += (ds - ds * scale) / 2;
-      dy += (ds - ds * scale) / 2;
-      ds *= scale;
-
-      // Rotate the halite to get a bit of randomness.
-      move(
-        bgCtx,
-        { x: dx, y: dy, width: ds, height: ds, angle: haliteRotations[pos] },
-        () => {
-          bgCtx.drawImage(bufferCanvas, sx, sy, ss, ss, 0, 0, ds, ds);
-        }
-      );
-    });
+    // Render the halite.
+    halite.forEach((cellHalite, pos) => renderHalite(bgCtx, pos, cellHalite, 500, 1, true));
```

Comment:
Could be worth introducing named constants for the main numbers in this file, it's a little tricky to follow what means what.

Reply:
I totally agree, it's pretty bad, I wanted to do that but I kept finding higher priority features to add and I needed to come to a stopping point eventually :)

Reply:
Totally fair, we can always clean up for the next version if we end up wanting to add more visual stuff.